api/rosetta/errors.go

```diff
@@ -76,6 +81,12 @@ func withDetail(key string, val interface{}) detailFunc {
 	}
 }
 
+func withAddress(key string, val flow.Address) detailFunc {
```

Comment:
Perhaps unnecessary to have strictly typed `detailFunc` for this type alone, but felt nicer to do it here. Could just use a regular `withDetail` call.

Reply:
I think it's fine :+1: